ml-agents/mlagents/trainers/torch/distributions.py

```diff
@@ -115,7 +115,7 @@ def log_prob(self, value):
         return torch.log(self.pdf(value) + EPSILON)
 
     def all_log_prob(self):
-        return torch.log(self.probs + EPSILON)
```

Comment:
Would add a comment saying that self.logits are already normalized due to the action masking.

Reply:
Why not modify the masking instead then ?